src/helpers/array.helper.ts

```diff
@@ -34,4 +37,14 @@ export class ArrayHelper {
     }
     return true;
   }
+
+  public static stableSort<T>(array: T[], comparator: (a: T, b: T) => Comparator): T[] {
```

Comment:
Dodałem nową funkcję do helpera, która sortuje tablice. Napiszę do niej testy. Używam w niej funkcji do porównywania, która znajduje się obecnie w komponencie [workers-tab.component](https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-frontend/blob/9785b9ce9e4d7e2ce1e7ff1d606a7097fcb49147/src/components/workers-page/workers-tab/workers-tab.component.tsx) Przydałoby się znaleźć dla niej lepsze miejsce. @Qwebeck

Reply:
Myślę, że `ArrayHelper` może być, ale ja bym napisał nową klasę `Comparer`. 
Przy tworzeniu ona będzie przejmować dwie listy i ona na razie będzie posiadać takie metody jak `stableSort` , `sort` i metody w stylu `descendingComparator`. I w ogóle będzie zawierać calą logikę sortowania co ty pisaleś
I dodalibyśmy ją na razie do folderu `helpers`. ( trochę później to zrefaktoryzujemy )

Reply:
Podoba mi się ta koncepcja z nowa klasa. Jednak nie tworzyłbym tej obiektów klasy, a dodałbym do niej metody statyczne i nazwał ComparatorHelper, żeby jednak trzymać się tej naszej konwencji. Nie rozumiem też po co miałaby przyjmować 2 listy skoro my tutaj sortujemy tylko jedną. Zaraz wrzucę tę poprawkę, dodatkowo uprościłem trochę ten kod, bo wcześniej na szybko go przekopiowywałem, a da się to zrobić w bardziej czytelny sposób :)

Reply:
Napisałem o dwóch listach, bo u mnie `Comparator` skojarzył się z czymś co porównuje. A do porównywania, przecież potrzebujemy dwóch elementów :D 
Ale masz rację xD